integration/awskms/src/aws_kms_aead_test.rs

```diff
@@ -91,3 +91,21 @@ fn test_basic_aead_without_additional_data() {
         }
     }
 }
+
+#[test]
+fn test_aead_with_invalid_key_fail() {
+    init();
+    let key_uri = "aws-kms://arn:aws:kms:us-east-2:1234567:key/aaaaa-bbbb-cccc-dddd-eeeee";
+    let client = crate::AwsClient::new(key_uri).unwrap();
+    let aead = client.get_aead(key_uri).unwrap();
+
+    // Not a valid key URI so everything will fail.
+    let result = aead.encrypt(b"data", b"aad");
+    tink_testutil::expect_err(result, "request failed");
+    let result = aead.encrypt(b"data", b"");
+    tink_testutil::expect_err(result, "request failed");
+    let result = aead.decrypt(b"data", b"aad");
```

Comment:
Lines 107-110 seem to be exact copies of 103-106. Is this intentional?

Reply:
One pair is `encrypt`, one is `decrypt`.

Reply:
Ah, I missed that part. :)